wire-ios/Wire-iOS/Sources/UserInterface/Collections/AssetCollectionWrapper.swift

```diff
@@ -55,19 +55,19 @@ extension AssetCollectionMulticastDelegate: AssetCollectionDelegate {
 }
 
 final class AssetCollectionWrapper: NSObject {
-    let conversation: ConversationLike
```

Comment:
Why do we need to change the type here? if I remember correctly we would like to use `ConversationLike` whenever is possible.

Reply:
typealias **GroupDetailsConversationType** = GroupDetailsConversation & **Conversation**
typealias Conversation = **ConversationLike** & SwiftConversationLike

I changed it to `GroupDetailsConversationType` because I need to use properties from the `GroupDetailsConversation` protocol.